integrations/ingress-nginx/documentation.yaml

```diff
@@ -0,0 +1,90 @@
+exporter_type: included
+app_name_short: Ingress NGINX
+app_name: {{app_name_short}}
+app_site_name: Ingress NGINX
```

Comment:
It looks like the title of that web page is "NGINX Ingress Controller"

(would be nice if they were consistent in their names! I've seen "Ingress NGINX Controller", "NGINX Ingress Controller" and "Ingress-NGINX controller"!)

Reply:
It is confusing. NGINX Ingress Controller is not the same as Ingress NGINX.  Ingress NGINX is the official ingress solution for Kubernetes, but other controllers exist such as haproxy and nginx ingress (from the nginx company, now F5). The core software is the same, but the featureset and configuration can differ. F5 offers paid support and features for NGINX Ingress (I believe), while Ingress NGINX is purely an open source solution provided by the Kubernetes project.

- Ingress NGINX: https://kubernetes.github.io/ingress-nginx
- NGINX Ingress: https://docs.nginx.com/nginx-ingress-controller/

This sheet has a nice breakdown, and turns out there are several other nginx controllers available from various vendors. https://docs.google.com/spreadsheets/d/191WWNpjJ2za6-nbG4ZoUMXMpUK8KlCIosvQB0f-oq3k/edit#gid=907731238

Reply:
God, that is confusing. This value was just supposed to capture the title of the web page we're linking to (so they know they are in the right place). I hope that someone looking at this already knows what this is, so maybe it's only confusing to me. Would it help to make this "Kubernetes Ingress NGINX"? (I guess I don't understand why this page, a Kubernetes page, is titled NGINX Ingress Controller -- maybe that's the problem?) But I will defer to your judgement on this, because I'm hopelessly confused! Thanks for the background.

Reply:
Also, does this mean we shouldn't change "app_name" (line 3) or "exporter_name" (line 6) to append "Controller"?? (That suggestion was inspired by the fact that the page at this link kept referring to "controller". But if that's not what we have here? (Again, if this was a stupid suggestion, please revert it -- as I said, I'm hopelessly confused about this at this point.)

Reply:
I think keeping "controller" makes sense. Yes, the ingress world is very confusing.